lightly/cli/config/lightly-serve.yaml

```diff
@@ -0,0 +1,10 @@
+
+input_dir: ''     # Path to the input directory.
+lightly_dir: ''   # Path to the lightly directory.
```

Comment:
Up for discussion.

I called the mount points in the worker `input_datasource` and `lightly_datasource`. I found it more descriptive. What do you think about calling it the same for lightly-serve?

Reply:
Hm, maybe that's not accurate. It is more `input_root` and `lightly_root`, the datasources are a paths under them.

Reply:
Note: We decided on `input_mount` and `lightly_mount`.